### ISSUE TITLE
[Link and button labels] Your content doesn't use correct and consistent labels across links, entry points, navigation components, and H1s. (08.02.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#46637]
#### VFS team name
MOVE

#### VFS product name
1095B Tax Form

#### VFS feature name
1095B Tax Form - accessibility version

#### Point of Contact/Reviewers
Cheryl Evans (@CherylEvans) - Content

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Your content doesn't use correct and consistent labels across links, entry points, navigation components, and H1s.

### Issue Details
Most download buttons contain text: "Download your 1095-B tax form".  One button contains text: "Download form".  All of the buttons should be consistent, as they all are referring to downloading tax form 1095-B.

### Link, screenshot or steps to recreate
https://drive.google.com/drive/folders/1scKdZxUXDIYJbpP5sNiVuLIK3Is2QHKa

### VA.gov Experience Standard
[Category Number 08, Issue Number 02](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.6_AA

---

### Platform Recommendation
Make all text on download buttons consistent.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by tameracorson** [comment]
@shiragoodman designs in code are correct.   Will amend designs in attachments to main ticket by COB and ping you back.

**Comment by tameracorson** [comment]
@shiragoodman Here are the amended Sketch designs that reflect what is in code regarding Download button content https://www.sketch.com/s/2f061f33-42de-4d8f-83ad-40286632ba36/p/D703716B-FAF8-478D-9CC4-7B9FB0FE4BE5

**Comment by shiragoodman** [comment]
@CherylEvans could you take a look?

**Comment by tameracorson** [comment]
@CarlDickerson Linking you here for any feedback on the sketch updates by Cheryl

**Comment by CherylEvans** [comment]
@shiragoodman I used the link provided, and was directed to make an account in Sketch, which I did.  When I logged into the account, and tried to go to the provided link, I was told I didn't have access.  I was given the option of requesting access, which I did.  I will let you know if I have additional issues.

**Comment by CherylEvans** [comment]
@tameracorson @shiragoodman Just reviewed the designs.  The mobile version looks great.  I assume the use of the color Green as opposed to Blue, is used to differentiate between PDF and non-PDF, correct?  Looks great!!  Thank you for providing the new screenshots.
